Crashlytics/Crashlytics/Handlers/FIRCLSException.mm

```diff
@@ -271,6 +279,7 @@ void FIRCLSExceptionRecord(FIRCLSExceptionType type,
 
   // Create new report and copy into it the current state of custom keys and log and the sdk.log,
   // binary_images.clsrecord, and metadata.clsrecord files.
+  // also copy rollouts.clsrecord if applicable.
```

Comment:
curious question: is this happening implicitly? I don't see any change to implement it (at least I think)

Reply:
Ah if you the the line below, this is actually copy all the file from the one report directory to another. So if the original report contain rollouts then it will be copy to another report directory using the existing logic.
```
  BOOL copied = [fileManager.underlyingFileManager copyItemAtPath:currentReportPath
                                                           toPath:newReportPath
                                                            error:&error];
```